README.md

```diff
@@ -269,10 +281,18 @@ if err != nil {
 }
 ```
 
+<!-- GO EXAMPLE
+}
+-->
+
 ### Querying the Presence History
 
+<!-- GO EXAMPLE
+{
+-->
+
 ```go
-page, err = channel.Presence.History(ctx, nil)
+page, err := channel.Presence.History(ctx, nil)
 for ; err == nil && page != nil; page, err = page.Next(ctx) {
 	for _, presence := range page.PresenceMessages() {
```

Comment:
if it's a readme do we need to ignore unused param 🤔 .. I mean it can be there for the sake of it ... like just to get gist of what it exactly returns

Reply:
This is outside the scope of this PR.